go/vt/vttest/local_cluster.go

```diff
@@ -497,9 +499,29 @@ func (db *LocalCluster) loadSchema(shouldRunDatabaseMigrations bool) error {
 	return nil
 }
 
+func (db *LocalCluster) createVTSchema() error {
```

Comment:
May be we should name this to syncVTDatabase, in order to align with schema/engine.go

Reply:
It will always create the `_vt` schema instead of syncing it and it matches the other call `createDatabase()` called earlier.